src/recipes/sergeant-pepper.json

```
@@ -0,0 +1,75 @@
+{
+    "name": "<Sergeant Pepper>",
```

```suggestion
    "name": "Sergeant Pepper",
```

You can remove these symbols for all fields as these were just placeholders as an example. Thanks.